### PR TITLE
fix minor typos (missplling & grammar)

### PR DESCRIPTION
**Pull request by mohamedelraghy**
## Related Issues

-

### CODE REVIEW

File: docs/guides/faq.md [hunk 1]
```diff
@@ -73,7 +73,8 @@ class Cat {
 
 ### Why is `_id` `unknown`?
 
-It is very likely that your class is just empty, and typescript somehow does not correctly match that and treats it like a generic object.
+Your class is likely just empty, and typescript somehow does not correctly match that and treats it like a generic object.
+
 
 Example:
```
Comment (hasezoey) [inline docs/guides/faq.md:79]:
```suggestion
Your class is likely just empty, and typescript somehow does not correctly match that and treats it like a generic object.

Example:
```

File: website/README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 This website is built using [Docusaurus 2](https://v2.docusaurus.io/), a modern static website generator.
 
-Note: `yarn` is required, otherwise some plugins don't work
+Note: `yarn` is required otherwise, some plugins don't work
```
Comment (hasezoey) [inline website/README.md:3]: is this really correct?

**Review by hasezoey — COMMENTED** [review]
Thanks, mostly looks ok

File: website/README.md (hunk 2 above)
Comment (mohamedelraghy) [reply website/README.md:3]: yes, I Checked with a Grammarly tool

Comment (mohamedelraghy) [reply website/README.md:3]: http://www.centralsoutherntierraen.org/protected/documents/comma_rules2.pdf

you could check rule 10 in the Comma_Rules

**Pull request merged**